### PR TITLE
feat(pie-input): DSW-1582 add a value property to input

### PR DESCRIPTION
**Pull request by jamieomaguire**
## Tested in:
- PIE Aperture NextJS (SSR Enabled)

## Note:
- Vanilla form integration is coming in a separate PR

---
"@justeattakeaway/pie-input": minor
---

[Added] - `value` property to input component

---
"pie-storybook": minor
---

[Added] - value prop to pie-input stories

### CODE REVIEW

File: packages/components/pie-input/custom-elements.json [hunk 1]
```diff
@@ -54,6 +71,37 @@
...
+                            },
+                            "privacy": "public",
+                            "default": "''",
+                            "attribute": "value"
+                        },
+                        {
+                            "kind": "field",
+                            "name": "handleInput",
+                            "privacy": "private",
+                            "description": "Handles the input event and dispatches a custom event with the input event data and input value.",
```
Comment (siggerzz) [inline packages/components/pie-input/custom-elements.json:89]: ⭐ for the JSDoc!

**Review by siggerzz — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**